### PR TITLE
Rearranged instruction, Changed position of the instruction

### LINKED ISSUE #27722: Using git in the real world: convert step to a Note card and not an instruction
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

The  notification is placed as if it is a step but it is more of a note or something you hsould keep in mind so there is no need for numbering and the styling should be somewhat of a blockquote

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/javascript-using-git-in-the-real-world#:~:text=Note%20that%20a%20git%20fetch%20upstream%20followed%20by%20a%20git%20merge%20upstream/some_branch%20is%20the%20EXACT%20same%20thing%20as%20doing%20a%20git%20pull%20upstream%20some_branch.%20We%20prefer%20to%20split%20it%20up%20here%20so%20that%20we%20can%20explicitly%20walk%20through%20the%20steps.

### (Optional) Discord Name

_No response_

### (Optional) Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by Mtendekuyokwa19**
## Because

I rearranged the postion of the instruction to the bottom because it was more of a notice and less of a step to follow.
It described extra information or a justification of a previous step.

## This PR

- Postion of instruction
- markdown style of instruction

## Issue

Closes #27722

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: git/intermediate_git/using_git_in_the_real_world.md [hunk 1]
```diff
@@ -41,11 +41,14 @@ git remote add upstream git@github.com:TheOdinProject/curriculum.git
...
+5. You may have merge conflicts... resolve those using the skills you learned in [the Deeper Look at Git lesson](https://www.theodinproject.com/lessons/ruby-a-deeper-look-at-git)! ([JS Course Link](https://www.theodinproject.com/lessons/javascript-a-deeper-look-at-git))
+
+
+ >Note that a `git fetch upstream` followed by a `git merge upstream/some_branch` is the EXACT same thing as doing a `git pull upstream some_branch`. We prefer to split it up here so that we can explicitly walk through the steps.
```
Comment (MaoShizhong) [inline git/intermediate_git/using_git_in_the_real_world.md:51]: We should use a [note box](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#note-boxes) for this part, and make sure we only have one blank line above, not two.

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
Thanks for opening this PR @Mtendekuyokwa19.

As per the [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#lazy-numbering), we need to use lazy numbering to let markdown figure out the right numbers automatically, so could you change these back? And also address the comment below.

Besides these two things, you don't need to fix any of the other lint errors (default section content and descriptive link).

**Comment by Mtendekuyokwa19** [comment]
I have made the updates but the lint test above isnt passing the tests

File: git/intermediate_git/using_git_in_the_real_world.md [hunk 2]
```diff
@@ -43,9 +43,15 @@ We've got one main branch -- `main`. `main` is for production-ready code. Any co
 1. Create a new feature branch for whatever feature you want to build, and add commits following the same practices that you learned about in the [branching section of our Revisiting Rock Paper Scissors lesson](https://www.theodinproject.com/lessons/foundations-revisiting-rock-paper-scissors#using-branches).
 1. When you're done with your feature, odds are that someone has made changes to the upstream repository in the meantime. That means that your `main` branch is probably out of date. Fetch the most updated copy using `git fetch upstream`.
 1. Now merge the upstream's changes into your local version of `main` using `git merge`. Specifically, you'll first want to make sure you're on your `main` branch using `git checkout main` and then `git merge upstream/main` to merge in those upstream changes that we just fetched.
-1. Note that a `git fetch upstream` followed by a `git merge upstream/some_branch` is the EXACT same thing as doing a `git pull upstream some_branch`. We prefer to split it up here so that we can explicitly walk through the steps.
+
```
Comment (MaoShizhong) [inline git/intermediate_git/using_git_in_the_real_world.md:46]:
```suggestion
```
Looks great! We just need to get rid of this rogue blank line now to keep the file tidy.

**Review by MaoShizhong — CHANGES_REQUESTED** [review]

**Comment by MaoShizhong** [comment]
You can ignore the rest of the lint errors - they're unrelated to this PR and something to be fixed in a different one.

**Comment by Mtendekuyokwa19** [comment]
so everything is okay right ?

**Comment by MaoShizhong** [comment]
> so everything is okay right ?

Just need to remove the blank line on line 46 as shown above, but after that yes, should be good to go.

**Comment by Mtendekuyokwa19** [comment]
I have removed the blank line on 46

**Review by MaoShizhong — APPROVED** [review]
Thanks for getting this done!

**Pull request merged**